edweiss-firebase/functions/src/functions/todolist/updateTodo.ts

```diff
@@ -31,6 +31,14 @@ export const updateTodo = onAuthentifiedCall(Functions.updateTodo, async (userId
     if (todo == undefined)
         return fail("firebase_error");
 
+    const todoCollection = CollectionOf<Todo>(`users/${userId}/todos`);
+
+    const existingTodos = await todoCollection.where('name', '==', args.name).get();
+
+    if (!existingTodos.empty) {
```

Comment:
same for here again, check please if it works as just if(exisitingTodos)

Reply:
So for this case, it for the `updateTodo` cloud function. The only thing to do is to check if the filtered collection do not already contains a `Todo` with this name. Once you check this you can either exit on `fail` as there's a duplicate todo or just update the `Todo`.
Make more sens ?

Reply:
yes thank you so much